cnames_active.js

```diff
@@ -3338,6 +3338,7 @@ var cnames_active = {
   "tui-nuxt": "hosting.gitbook.com", // noCF
   "tulsa": "thatguysam.github.io/tulsa.js",
   "turali": "caglarturali.github.io",
+  "tusk": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "tusk": "cname.vercel-dns.com", // noCF
```
As you're using Vercel, you'll need the Cloudflare proxy disabled